doc/source/ray-air/quickstart.md

```diff
@@ -0,0 +1,302 @@
+# Ray AI Runtime
+
+Ray AI Runtime (AIR) is an open-source toolkit for building end-to-end ML applications. By leveraging Ray and its library ecosystem, it brings scalability and programmability to ML platforms.
+
+The main focuses of the Ray AI Runtime:
+
+* Ray AIR focuses on providing the compute layer for ML workloads.
+* It is designed to interoperate with other systems for storage and metadata needs.
+
+```{image} images/ecosystem.svg
```

Comment:
The diagram could use some improving, maybe get some UX design help?

Reply:
+1